docs/user-guide/migration.md

```diff
@@ -0,0 +1,65 @@
+# Migration to apiVersion networking.k8s.io/v1
+
+- Please read this [official blog on deprecated ingress api versions](https://kubernetes.io/blog/2021/07/26/update-with-ingress-nginx/) If you are using ingress objects in your pre K8s v1.22 cluster, and you upgrade to K8s v1.22, then this document may be relevant to you.
+
+- Please read this [official documentation on the IngressClass object](https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-class)
+
+## What is ingressClass ?
+
```

Comment:
I would add an statement tl;dr of what is IngressClass, like
"On clusters with more than one Ingress controller, the controllers must be aware of what Ingress they must serve. Ingress class is the way to let the controller know about that"